src/main/java/com/zaxxer/nuprocess/internal/BasePosixProcess.java

```diff
@@ -138,18 +138,6 @@ protected BasePosixProcess(NuProcessHandler processListener)
     */
    public abstract void run(List<String> command, String[] environment, Path cwd);
 
-   /**
-    * Check the launched process and return {@code true} if launch was successful,
-    * or {@code false} if there was an error in launch.
-    *
-    * @return {@code true} on success, {@code false} on failure
-    */
-   protected boolean checkLaunch()
```

Comment:
Only `OsxProcess` was ever calling this, so it doesn't need to be in the base class